x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/datatiers/DataTierUsageFixtures.java

```diff
@@ -79,7 +79,13 @@ static NodeIndicesStats buildNodeIndicesStats(RoutingNode routingNode, long byte
             IndexShardStats shardStats = new IndexShardStats(shardId, new ShardStats[] { shardStat });
             indexStats.computeIfAbsent(shardId.getIndex(), k -> new ArrayList<>()).add(shardStats);
         }
-        return new NodeIndicesStats(COMMON_STATS, Map.of(), indexStats, true);
+        return new NodeIndicesStats(
+            COMMON_STATS,
+            Map.of(),
+            indexStats,
+            /* projectsByIndex is only used for rendering as XContent, which not needed here */ Map.of(),
```

Comment:
Teensiest nit ever:
```suggestion
            // projectsByIndex is only used for rendering as XContent, which not needed here
            Map.of(),
```
The syntax you used is not very common here (AFAIK) and it took me longer than I'm willing to admit to realize the parameter came after the comment.